src/main/java/frc/robot/commands/CmdShooterBedSetBedAngle.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.shooter.ShooterBed;
+
+public class CmdShooterBedSetBedAngle extends Command
+{
+    private ShooterBed _shooterBed;
+    private Rotation2d _bedAngle;
+
+    public CmdShooterBedSetBedAngle(ShooterBed shooterBed, Rotation2d bedAngle)
```

Comment:
We'll only ever realistically instantiate this command with a bed angle in degrees. I'd like for the subsystem to accept the desired angle as a `Rotation2d` for flexibility, but I think this command can probably take in the angle as a double representing degrees, and convert it to a `Rotation2d` in the constructor